src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferencesSearchEngine_FindReferencesInDocuments.cs

```diff
@@ -180,7 +179,7 @@ async ValueTask<bool> HasInheritanceRelationshipSingleAsync(ISymbol searchSymbol
             // Counter-intuitive, but if these are matching symbols, they do *not* have an inheritance relationship.
             // We do *not* want to report these as they would have been found in the original call to the finders in
             // PerformSearchInTextSpanAsync.
-            if (await SymbolFinder.OriginalSymbolsMatchAsync(_solution, searchSymbol, candidate, cancellationToken).ConfigureAwait(false))
+            if (SymbolFinder.OriginalSymbolsMatch(_solution, searchSymbol, candidate))
```

Comment:
OriginalSymbolsMatchAsync used to be async, but i was able to make it async based on a change to the workspace.  thsi will be explained later on.